algebra-core/src/curves/bucketed_add.rs

```diff
@@ -0,0 +1,256 @@
+use crate::{cfg_iter_mut, curves::BatchGroupArithmeticSlice, log2, AffineCurve, Vec};
+
+#[cfg(feature = "parallel")]
+use rayon::prelude::*;
+
+#[cfg(feature = "std")]
+use std::collections::HashMap;
+
+const RATIO_MULTIPLIER: usize = 2;
+const BATCH_ADD_SIZE: usize = 4096;
```

Comment:
There was another place that references 4096, could they refer to the same constant?

Reply:
I've made it a common constant in batch arith.rs and add some comments